static/usr/lib/core/handle-writable-paths

```diff
@@ -181,6 +181,7 @@ main()
 		echo "/dev/root / rootfs defaults,ro 0 0" >> "$fstab"
 		handle_writable_paths "$writable_paths" "$fstab"
 		echo "/run/mnt/ubuntu-seed /var/lib/snapd/seed none bind,ro 0 0" >> "$fstab"
+		echo "/run/mnt/ubuntu-save /var/lib/snapd/save none bind,rw 0 0" >> "$fstab"
```

Comment:
ubuntu-save is optional if encryption is not enabled

we also need the same change in core20?

Reply:
> ubuntu-save is optional if encryption is not enabled

I suppose it is always created. But if it is removed, it should still work. Then maybe the tests should try removing the partition and see if still boots.

Without `nofail`, it should in theory fail to boot.

Reply:
> 

Interesting point about `nofail`, I believe it would make sense to put that

Reply:
Well that will continue the boot even if the device is there but the mount itself fail. My preference is to add that line only if the /run/mnt/ubuntu-save exists